lib/channel_model.js

```diff
@@ -179,6 +179,7 @@ class Channel extends BaseChannel {
       this.unregisterConsumer(consumerTag);
       return ok.fields;
     });
+    return ok
```

Comment:
Returning ok looks like a change in behaviour. Was this previously an omission or just made to appease a linting rule? 
What does the Basic Cancel RPC return?
Were the api docs updated?